### PR TITLE
Fix faulty indentation

### PR DESCRIPTION
**Pull request by teoljungberg**
- Remove unused variable from GithubApiSpec

### CODE REVIEW

File: spec/lib/github_api_spec.rb [hunk 1]
```diff
@@ -181,59 +181,59 @@
...
     end
+  end
 
-    it "returns user's teams" do
-      teams = ["thoughtbot"]
-      client = double(user_teams: teams)
-      allow(Octokit::Client).to receive(:new).and_return(client)
-      hound_token = ENV["HOUND_GITHUB_TOKEN"]
-      api = GithubApi.new(hound_token)
+  it "returns user's teams" do
```
Comment (gylaz) [inline spec/lib/github_api_spec.rb]: Do you mind wrapping this into `describe "#user_teams" do`?

**Comment by gylaz** [comment]
:+1:

File: spec/lib/github_api_spec.rb (hunk 1 above)
Comment (teoljungberg) [inline spec/lib/github_api_spec.rb]: Fixed and force-pushed

**Comment by teoljungberg** [comment]
Thanks @gylaz

**Pull request merged**